presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -357,29 +359,148 @@ public void testQuantifiedComparisonNotEqualAny()
         );
     }
 
+    @Test
```

Comment:
Can we move these into a new test class? TestLogicalPlanner is getting big as it stands and this group of tests seems to share a common them.